release/src/main/scripts/mass_comment.py

```diff
@@ -0,0 +1,141 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Script for mass-commenting Jenkins test triggers on a Beam PR."""
+
+import itertools
+import os
+import socket
+import sys
+import time
+import traceback
+import re
+import requests
+from datetime import datetime
+
+
+COMMENTS_TO_ADD=[
+  "Run Go PostCommit",
+  "Run Java PostCommit",
+  "Run Java PortabilityApi PostCommit",
+  "Run Java Flink PortableValidatesRunner Batch",
+  "Run Java Flink PortableValidatesRunner Streaming",
+  "Run Apex ValidatesRunner",
+  "Run Dataflow ValidatesRunner",
+  "Run Flink ValidatesRunner",
+  "Run Gearpump ValidatesRunner",
+  "Run Dataflow PortabilityApi ValidatesRunner",
+  "Run Samza ValidatesRunner",
+  "Run Spark ValidatesRunner",
+  "Run Python Dataflow ValidatesContainer",
+  "Run Python Dataflow ValidatesRunner",
+  "Run Python 3.5 Flink ValidatesRunner",
+  "Run Python 2 PostCommit",
+  "Run Python 3.5 PostCommit",
+  "Run SQL PostCommit",
+  "Run Go PreCommit",
+  "Run Java PreCommit",
+  "Run Java_Examples_Dataflow PreCommit",
+  "Run JavaPortabilityApi PreCommit",
+  "Run Portable_Python PreCommit",
+  "Run PythonLint PreCommit",
+  "Run Python PreCommit",
+  "Run Python DockerBuild PreCommit"
+]
```

Comment:
Should we also remove the duplicate list from [verify_release_build.sh](https://github.com/apache/beam/blob/master/release/src/main/scripts/verify_release_build.sh#L43) as part of this PR?

I'm not really clear on where this list comes from. Is the goal to launch every single jenkins job?

Reply:
Probably we should just call mass_comment.py from verify_release_build.sh.

The goal of the list is to run all the Jenkins jobs that might provide relevant signal for the release. My understanding is, since we for the most part only run Jenkins cron jobs on head, creating a dummy PR is the best way to test the release branch. I bet it's possible to configure Jenkins cron jobs to run against the release branch instead, but not sure if that would be worth the effort or expense.

Also, this list does look out of date (Python 3.6 and 3.7 are conspicuously missing, for example). I'll update it later.

Reply:
There are also some jobs that no longer exist in this list (removed in https://github.com/apache/beam/pull/11635)

Reply:
I made some updates, PTAL